### PR TITLE
Adding an option for downloading the dataset

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

File: src/Microsoft.DotNet.Github.IssueLabeler/CommandLineOptions.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    internal class CommandLineOptions
+    {
+        [Option('e', "EndIndex", Required = true, HelpText = "Ending Github issue number")]
+        public int EndIndex { get; set; }
+
+        [Option('o', "OutputFile", Required = true, HelpText = ".tsv Output File Path")]
+        public string Output { get; set; }
+
+        [Option('u', "RepoOwner", Required = true, HelpText = "Repository owner")]
```
Comment (danmosemsft) [inline src/Microsoft.DotNet.Github.IssueLabeler/CommandLineOptions.cs]: u ? 😄

File: src/Microsoft.DotNet.Github.IssueLabeler/GithubIssueDownloader.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                        sb.Clear();
+                    }
+                }
+                catch (Exception)
+                {
+                    Console.WriteLine($"Issue {i} does not exist");
+                }
+            }
+
+            File.AppendAllText(_outputFile, sb.ToString());
```
Comment (danmosemsft) [inline src/Microsoft.DotNet.Github.IssueLabeler/GithubIssueDownloader.cs]: I guess _outputFile needs to be empty before you start.

**Comment by Anipik** [comment]
@danmosemsft can you another go. I have to make a couple more changes in order to consume the auto ml model and ml.net 1.0

**Comment by Anipik** [comment]
I have deployed and tested the code though

**Pull request merged**